Libraries/Opc.Ua.Gds.Client.Common/ServerCapabilities.cs

```diff
@@ -232,7 +232,7 @@ public string ToString(string format, IFormatProvider formatProvider)
         public const string LocalDiscoveryServer = "LDS";
 
         /// <summary>
-        /// The server supports the data integration (DI) information model.
+        /// The server supports the device integration (DI) information model.
```

Comment:
Ist DI device Integration? I need to look that up

Reply:
Yes, it is, see https://reference.opcfoundation.org/DI/v104/docs/3.2. At least that was the origin. Today most of the times the spec / nodeset is referred as "OPC UA for Devices" or even shorter "Devices".